components/style/inline.rs

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use computed_values::white_space;
+
+pub trait WhitespaceMethods {
```

Comment:
Ah, sorry for the misunderstanding, my earlier message wasn’t clear and didn’t have it in mind anymore when I replied here.

Since Ms2ger mentioned replacing something from `components/layout/fragment.rs`, I was assuming that it would stay in the layout crate. If you want to define in the layout crates methods for a type defined in the style crate, then you’d need a trait. And if it’s related to inline layout, it would make sense to have it in `components/layout/inline.rs`.

But looking now at what these methods actually do, it’s not a complex layout algorithm that particularly belongs in the layout crate. I think having it in the style crate is fine.

So, yeah, please make it an inherent impl without a trait, next to the definition of the white space property in `properties.mako.rs`. Thanks!